src/csvcubed/utils/qb/validation/observations.py

```diff
@@ -195,4 +328,83 @@ def _validate_pivoted_shape_cube(
             )
         )
 
+    else:
+        # There are multiple obs val columns and no measure columns, so it looks like the user is aiming for a
+        # pivoted shape. We assume the user wants a pivoted shape, so let them know that they're missing measures
+        # against some of their obs val columns.
+        "Location, Average Income (meas defined), Value (no meas defined)"
+        "Birmingham, 22, 45.6"
+        errors.append(
+            NoMeasuresDefinedError(
+                additional_explanation="Data apears to attempt the pivoted shape, however observation value column(s) have been found without a measure linked."
+            )
+        )
+
+    return errors
+
+
+def _get_error_when_cube_only_contains_obs_val_cols_without_measures(
+    cube: Cube, obs_val_columns_without_measure: list[QbColumn[QbObservationValue]]
+) -> List[ValidationError]:
+    errors: List[ValidationError] = []
+
+    measure_columns = get_columns_of_dsd_type(cube, QbMultiMeasureDimension)
+    if any(measure_columns):
+        """
+        Example of an input that might result in a user ending up here:
+
+        Location, Income (no meas defined), Average Age (no meas defined), Income Measure, Average Age Measure
+        Birmingham, 22, 45.6, Average Income, Average Age
+        """
+        # There are mutliple obs val columns defined without measures, and at least one measure column defined.
+        # This is an erroneous hybrid between standard and pivoted shape.
+        errors.append(
+            HybridShapeError(
+                obs_val_columns_without_measure,
+                measure_columns,
+            )
+        )
+    else:
+        # There are multiple obs val columns defined without measures, so it looks like the user is aiming for a
+        # pivoted shape. We assume the user wants a pivoted shape, so let them know that they're missing measures
+        # against some of their obs val columns.
+        "Location, Income (no meas defined), Average Age (no meas defined)"
+        "Birmingham, 22, 45.6"
+        errors.append(
+            NoMeasuresDefinedError(
+                additional_explanation="Data apears to attempt the pivoted shape, however observation value columns have been found without a measure linked."
+            )
+        )
+
+    return errors
+
+
+def _get_errors_for_standard_shape_cube(
+    cube: Cube,
+    obs_val_columns_without_measure: list[QbColumn[QbObservationValue]],
+    multi_units_columns: List[QbColumn[QbMultiUnits]],
+) -> List[ValidationError]:
+    errors: List[ValidationError] = []
+
+    obs_val_column = obs_val_columns_without_measure[0]
+    errors += _validate_observation_value(obs_val_column, multi_units_columns)
+    errors += _validate_standard_shape_cube(cube)
+
+    return errors
+
+
+def _get_errors_when_cube_only_contains_obs_val_cols_with_measures(
```

Comment:
Can you call this `validate` rather than `get_errors`? We don't necessarily return errors from here if everything is in order. I believe the other functions are okay since they will always return errors.

Reply:
Renamed